tests/knative-serving/knative_serving_test.go

```diff
@@ -77,6 +84,16 @@ func TestKnativeServingAcceptance(t *testing.T) {
 		},
 	}
 
+	// Cleanup test resources gracefully when an interrupt signal is received.
+	interrupted := cleanupOnInterrupt(t, interruptSignals, func() { deleteService(t, serviceClient, svc.Name) })
+	defer close(interrupted)
+
+	// If the Knative service already exists, delete it to make sure
+	// the new object to be created has the correct structure and data.
+	if _, err = serviceClient.Get(svcName, meta.GetOptions{}); err == nil {
+		deleteService(t, serviceClient, svc.Name)
```

Comment:
we should wait until the object is gone in this case. Otherwise the next call where we create the object again might fail

Reply:
Done.